### PR TITLE
Add instruction to extractable strings

### PR DESCRIPTION
**Pull request by berroar**
### What is the context of this PR?
Adds the instruction field as an extractable string for answers

### How to review 
Check that instruction fields are extracted for translation

### CODE REVIEW

**Comment by berroar** [comment]
[Allow instruction and label for Checkbox answers (2 SP)](https://trello.com/c/D9VWhami/4838-allow-instruction-and-label-for-checkbox-answers-2-sp)

File: eq_translations/extractable_strings.py [hunk 1]
```diff
@@ -156,6 +156,11 @@
         "description": "Answer",
         "context": "Question",
     },
+    {
+        "json_path": "$..answers[*].instruction",
+        "description": "Answer",
```
Comment (ajmaddaford) [inline eq_translations/extractable_strings.py:161]: This should be more descriptive, something like "Checkbox answer instruction"

**Review by ajmaddaford — COMMENTED** [review]
Ideally a checkbox instruction would be added to the test language schema so that the extraction can be tested. This was done recently in https://github.com/ONSdigital/eq-translations/commit/5229ca188faa0eec69f220dbf48530e44c697ef7.

File: eq_translations/extractable_strings.py [hunk 2]
```diff
@@ -156,6 +156,11 @@
         "description": "Answer",
         "context": "Question",
     },
+    {
+        "json_path": "$..answers[*].instruction",
+        "description": "Checkbox Answer Instruction",
```
Comment (ajmaddaford) [inline eq_translations/extractable_strings.py:161]: The casing should be sentence case to be consistent with the other descriptions.
```suggestion
        "description": "Checkbox answer instruction",
```

**Review by ajmaddaford — APPROVED** [review]

**Review by MebinAbraham — APPROVED** [review]

**Pull request merged**